main.go

```diff
@@ -154,17 +154,24 @@ func dateParamToEpoch(s string, d int64) int32 {
 	switch s {
 	case "now":
 		return int32(timeNow().Unix())
-	case "yesterday", "today", "tomorrow":
+	case "yesterday", "today", "tomorrow", "midnight", "noon", "teatime":
 		t := timeNow()
 		switch s {
 		case "yesterday":
-			t.AddDate(0, 0, -1)
-		case "tomorrow":
-			t.AddDate(0, 0, 1)
+			t = t.AddDate(0, 0, -1)
+		case "tomorrow", "midnight":
```

Comment:
Trying "midnight" in graphite I get the current date midnight [past] not tomorrow's midnight [future].